### ISSUE TITLE
No working on Catalina?

### ISSUE THREAD

**Issue by Boycottangent** [aCluelessDanny/UeberPlayer#9]
Catalina 10.15.17
UeberPlayer and Übersicht are both latest version. Install and refresh but I don't see widget on desktop. 

<img width="1000" alt="image" src="https://user-images.githubusercontent.com/23280916/94514329-7eee0000-0253-11eb-9ae7-4f2e0ee0198f.png">

**Comment by leoVolk** [comment]
Try creating a cache folder in the UeberPlayer.widget folder and refreshing. This worked for me.

**Comment by Boycottangent** [comment]
> Try creating a cache folder in the UeberPlayer.widget folder and refreshing. This worked for me.

Yeah, first I found that there's some character issues in my local files. Revising this problem and mutually creating a cache folder makes widget load successfully. But still without showing album artwork when using Apple Music as #7 :-(

<img width="258" alt="image" src="https://user-images.githubusercontent.com/23280916/94524860-c67d8780-0265-11eb-9452-533028e905d5.png">

BTW, the character issues I mentioned seem to be brought in latest version 1.2.0. 
https://github.com/aCluelessDanny/UeberPlayer/blob/da2c31b43ee7fb9e33df7cb2505403591faf7cf9/UeberPlayer.widget/lib/getTrack.scpt#L80

Maybe It should be `«class PNG »` instead of the strange `Çclass PNG È` @aCluelessDanny

**Comment by aCluelessDanny** [comment]
Huh, so for some reason, and I didn't notice this until just now, but I did actually write `«class PNG »`, but it seems like when I pushed the changes, GitHub changed it? I think something was up in the file encoding, & that's what may have caused issues for Apple Music specifically.

Also

> Try creating a cache folder in the UeberPlayer.widget folder and refreshing. This worked for me.

Duly noted because now when I look at the code again, I think I straight up forgot to create the directory when using Apple Music, ahah. Gonna fix that in the init script now
